jobs/run_travissync.groovy

```diff
@@ -0,0 +1,37 @@
+import util.Common
+Common.makeFolders(this)
+
+pipelineJob('travissync/travissync') {
+  description('Synchronize Travis CI with GitHub.')
+
+  properties {
+    rebuild {
+      autoRebuild()
+    }
+  }
+
+  label('travissync')
+  keepDependencies(true)
+  concurrentBuild()
+
+  def repo = SEED_JOB.scm.userRemoteConfigs.get(0).getUrl()
+  def ref  = SEED_JOB.scm.getBranches().get(0).getName()
+
+  triggers {
+    cron('17 * * * *')
```

Comment:
Using `H` in place of `17` is slightly preferable as it allows jenkins to choose a random skew.